### ISSUE TITLE
Format of Additional AMM data

### ISSUE THREAD

**Issue by ruidazeng** [samdevo/rebalance#8]
Are there other AMM data formats besides the Solana AMM v4 mentioned in README that we need to consider integrating? (e.g., Uniswap V2, V3, V4, Curve, Raydium, Orca, etc.) 

@ Jad @ Sam

**Comment by samdevo** [comment]
Yes - right now it's just Raydium V4 - ideally we'd have more DEXs on solana. If we wanted to expand to multiple chains (there is no uniswap on solana)

**Comment by ruidazeng** [comment]
> Yes - right now it's just Raydium V4 - ideally we'd have more DEXs on solana. If we wanted to expand to multiple chains (there is no uniswap on solana)

What is the data format of Raydium V4?

**Comment by ruidazeng** [comment]
> Yes - right now it's just Raydium V4 - ideally we'd have more DEXs on solana. If we wanted to expand to multiple chains (there is no uniswap on solana)

wait I was under the impression we would have uniswap too. how are uniswap DEXs accessed?

**Comment by samdevo** [comment]
> > Yes - right now it's just Raydium V4 - ideally we'd have more DEXs on solana. If we wanted to expand to multiple chains (there is no uniswap on solana)
> 
> 
> 
> wait I was under the impression we would have uniswap too. how are uniswap DEXs accessed?

Uniswap does not exist (yet) on Solan, unfortunately.
